src/main/java/com/onarandombox/MultiverseCore/MVWorld.java

```diff
@@ -516,22 +516,39 @@ public World getCBWorld() {
      */
     @Override
     public String getColoredWorldString() {
-        if (props.getAlias().length() == 0) {
+        if (!hasCustomName()) {
             props.setAlias(this.getName());
         }
 
-        if ((props.getColor() == null) || (props.getColor().getColor() == null)) {
+        if (!hasCustomColor()) {
             this.props.setColor(EnglishChatColor.WHITE);
         }
 
         StringBuilder nameBuilder = new StringBuilder().append(props.getColor().getColor());
-        if (props.getStyle().getColor() != null)
+        if (hasCustomStyle()) {
             nameBuilder.append(props.getStyle().getColor());
-        nameBuilder.append(props.getAlias()).append(ChatColor.WHITE).toString();
+        }
+        nameBuilder.append(props.getAlias()).append(ChatColor.WHITE);
 
         return nameBuilder.toString();
     }
 
+    private boolean hasCustomName() {
+        return props.getAlias() != null
+                && !props.getAlias().isEmpty()
```

Comment:
You don't need `#isEmpty` check here. It will always false if the `getAlias` is not equal to the world name.

Reply:
Actually, I made a mistake in the logic here. It should return false if the alias is **either** equal to the world name **or** empty.